src/simulator/src/listeners.js

```diff
@@ -571,10 +570,6 @@ export default function startListeners() {
     })
 
     zoomSliderListeners()
-    // setupLayoutModePanelListeners()
-    if (!embed) {
```

Comment:
why?

Reply:
```
   @mousedown="setupTimingListeners.plotAreaMouseDown"
   @mousemove="setupTimingListeners.plotAreaMouseMove"
   @mouseup="setupTimingListeners.plotAreaMouseUp"
```

now we dont need to setuplisteners using jquery as we have the above